assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```diff
@@ -147,4 +147,60 @@
 			margin-bottom: 0;
 		}
 	}
+
+	.googlesitekit-ad-blocking-recovery__step {
+
+		.googlesitekit-stepper__step-content {
+
+			p {
+				color: $c-surfaces-on-surface-variant;
+				font-size: $fs-body-lg;
+				letter-spacing: $ls-s;
+			}
+
+			p:first-child {
+				margin-top: 8px;
+			}
+		}
+	}
+
+	.googlesitekit-ad-blocking-recovery__step-place-tags {
+
+		.mdc-checkbox {
+			margin-left: -11px;
+		}
+
+		p.googlesitekit-ad-blocking-recovery__error-protection-tag-info {
+			font-size: $fs-body-md;
+			line-height: $lh-body-md;
+			margin: 0 0 20px 34px;
+		}
+	}
+
+	.googlesitekit-ad-blocking-recovery__step-create-message {
+
+		.googlesitekit-ad-blocking-recovery__create-message-footer-actions {
+			align-items: center;
+			display: flex;
+			flex-wrap: wrap;
+			gap: 26px;
```

Comment:
Similar to above, are there not any tokens we can use for these literal px values? Things like margin and gap should be standard so I wouldn't expect we'd need exceptions. If we did, let's document why those are.

Reply:
Great catch, I've updated it to use tokens wherever possible, thank you!